src/bar/BarChartWithErrors.ts

```diff
@@ -63,28 +63,18 @@ export default class BarChartWithErrors extends mixins(Bar) {
                             label += formatCallback(tooltipItem.yLabel);
                         }
 
-                        interface ErrorBars {
-                            [xLabel: string]: {
-                                minus: number
-                                plus: number
-                            }
-                        }
-
-                        interface ChartDataSetsWithErrors extends ChartDataSets {
-                            errorBars?: ErrorBars
-                        }
-
                         let minus = null
                         let plus = null
 
                         if (this.showErrors && tooltipItem.xLabel && typeof tooltipItem.datasetIndex !== "undefined" && data.datasets && data.datasets[tooltipItem.datasetIndex]) {
                             const errorBars = (data.datasets[tooltipItem.datasetIndex] as ChartDataSetsWithErrors).errorBars
-                            if (errorBars && errorBars[tooltipItem.xLabel]) {
-                                minus = errorBars[tooltipItem.xLabel].minus
-                                plus = errorBars[tooltipItem.xLabel].plus
+                            const xLabelErrorBars = errorBars ? errorBars[tooltipItem.xLabel] : null
+                            if (xLabelErrorBars) {
+                                minus = xLabelErrorBars.minus
+                                plus = xLabelErrorBars.plus
                             }
                         }
-                        if (minus && plus) {
+                        if ((minus || minus === 0) && (plus || plus === 0)) {
```

Comment:
According the ErrorBars interface, the plus and minus values cannot be null, so should be safe to more the label set inside the if statement.

Reply:
Right, but we just cooked up that ErrorBars interface locally in this branch, didn't we, so I don't think we can rely on that. What happens when we try to display some data in this  component which doesn't necessarily have uncertainty values defined?

The errorBars data actually gets generated from the input data prop to this component when its parent, `BarchartWithFilters` [uses the `getProcessedOutputData` util](https://github.com/reside-ic/vue-charts/blob/master/src/bar/BarChartWithFilters.vue#L43), which sets `errorBars` based on indicator metadata: 
https://github.com/reside-ic/vue-charts/blob/master/src/bar/utils.ts#L94

So actually, that data could certainly be null, or it could also be undefined if e.g. `error_high_column` isn't in the dataset. I think we do want to be resilient to that. So actually I think you were right enough with your original implementation (or could maybe make it more explicit with `if ((typeof minus === "number") && (typeof plus === "number")) {`. And we should also change `ErrorBars` types for minus and plus to be `number | null | undefined`